geomesa-fs/geomesa-fs-datastore/src/main/scala/org/locationtech/geomesa/fs/storage/converter/ConverterPartitionReader.scala

```diff
@@ -0,0 +1,75 @@
+/***********************************************************************
+ * Copyright (c) 2013-2017 Commonwealth Computer Research, Inc.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Apache License, Version 2.0
+ * which accompanies this distribution and is available at
+ * http://www.opensource.org/licenses/apache2.0.php.
+ ***********************************************************************/
+
+package org.locationtech.geomesa.fs.storage.converter
+
+import java.io.{File, FileInputStream}
+
+import com.typesafe.scalalogging.LazyLogging
+import org.apache.hadoop.fs.Path
+import org.locationtech.geomesa.convert.SimpleFeatureConverter
+import org.locationtech.geomesa.fs.storage.api.{FileSystemPartitionIterator, Partition}
+import org.opengis.feature.simple.{SimpleFeature, SimpleFeatureType}
+
+class ConverterPartitionReader(root: Path,
+                               partition: Partition,
+                               sft: SimpleFeatureType,
+                               converter: SimpleFeatureConverter[_],
+                               gtFilter: org.opengis.filter.Filter) extends FileSystemPartitionIterator with LazyLogging {
```

Comment:
Wait, I am asking about projection at query time.  I.e., if I want to get back only some of the fields in an SFT defined by the converter.